.github/workflows/PRLabeling.yml

```
@@ -15,22 +15,55 @@ jobs:
       - name: Fetch PR details
         id: pr_details
         run: |
-          # Safely extract additions and deletions
-          CHANGED_LINES=$(jq '.additions + .deletions' <<< "${{ toJson(github.event.pull_request) }}" || echo "0")
-          echo "Changed lines: $CHANGED_LINES"
+          # Fetch the pull request's list of changed files using GitHub API
+          CHANGED_FILES=$(curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files")
           
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)